code/modules/clothing/glasses/hud.dm

```diff
@@ -71,6 +71,14 @@
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
 	glass_colour_type = /datum/client_colour/glass_colour/green
 
+/obj/item/clothing/glasses/hud/diagnostic/sunglasses
+	name = "diagnostic sunglasses"
+	desc = "Sunglasses with a diagnostic HUD."
+	icon_state = "sunhuddiag"
+	item_state = "glasses"
+	flash_protect = 1

```

Comment:
TRUE

Reply:
the whole file is that way, you're welcome to spend 15 minutes fixing it

Reply:
It's your pr lad

Reply:
`flash_protect ` is an integer, not a boolean. I have seen it go up to 2 and as low as -1.

Reply:
oh yeah, what was I thinking. yeah, flash_protect has multiple settings, light flash protection and heavy flash protection